opm/material/components/C10.hpp

```diff
@@ -61,7 +61,7 @@ class C10 : public Component<Scalar, C10<Scalar> >
      * \brief The molar mass in \f$\mathrm{[kg/mol]}\f$ of molecular n-Decane.
      */
     static Scalar molarMass()
-    { return 0.0142;}
+    { return 0.142;}
```

Comment:
Checked with this site: https://www.webqc.org/molecular-weight-of-C10H22.html which claims 142.2817 g/mol. Not sure if that is actually a good place to find such numbers, but if it is I think we should add the extra digits. What would be the canonical source a chemistry expert would use?

Reply:
This should be no problem. But new question is whether we should add the extra digits to C1, CO2, etc.  Unless you insist strongly, I think correcting the order of the quantity should be what the PR is for.

Reply:
I agree, merging this now. But please keep this in mind and consider adding more digits to all of them.